install/kubernetes/cilium/charts/hubble-ui/templates/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.ui.enabled }}
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: hubble-ui
+spec:
+  replicas: {{ .Values.ui.replicas }}
+  selector:
+    matchLabels:
+      k8s-app: hubble-ui
+  template:
+    metadata:
+      labels:
+        k8s-app: hubble-ui
+    spec:
+      {{- if and .Values.priorityClassName (eq .Release.Namespace "kube-system") }}
```

Comment:
*If* we want to keep this in (I personally still don't see the value of it, but I see that this has been copied over from the original helm chart), we need to add a K8s version check here (as @aanm requested in a previous version of this PR).

Reply:
I agree with @gandro that marking hubble-ui as critical does not make sense.

Reply:
Ok, based on your feedback I have removed it.